dotnet/src/Connectors/Connectors.Memory.DuckDB/DuckDBMemoryStore.cs

```diff
@@ -43,14 +56,38 @@ public sealed class DuckDBMemoryStore : IMemoryStore, IDisposable
     }
 
     /// <summary>
-    /// Connect a in memory DuckDB database
+    /// Connect an in memory DuckDB database
+    /// </summary>
+    /// <param name="vectorSize">Embedding vector size.</param>
+    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
+    public static Task<DuckDBMemoryStore> ConnectAsync(int vectorSize,
+        CancellationToken cancellationToken = default)
+    {
+        return ConnectAsync(":memory:", vectorSize, cancellationToken);
+    }
+
+    /// <summary>
+    /// Connect an in memory DuckDB database
     /// </summary>
     /// <param name="connection">An already established connection.</param>
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
     public static async Task<DuckDBMemoryStore> ConnectAsync(DuckDBConnection connection,
         CancellationToken cancellationToken = default)
     {
-        var memoryStore = new DuckDBMemoryStore(connection);
+        var memoryStore = new DuckDBMemoryStore(connection, null);
```

Comment:
vector size is optional, don't need to enforce here.

```suggestion
        var memoryStore = new DuckDBMemoryStore(connection);
```